package.json

```diff
@@ -36,6 +37,11 @@
     "@commitlint/cli": "^7.2.1",
     "@commitlint/config-angular": "^7.1.2",
     "@commitlint/travis-cli": "^7.2.1",
+    "@feature-hub/rcgen": "^0.0.0",
+    "@rcgen/cli": "^0.8.0",
+    "@rcgen/core": "^0.8.0",
+    "@rcgen/filetypes": "^0.8.0",
+    "@rcgen/patchers": "^0.8.0",
```

Comment:
Do we really need all of those deps here and in the package?

Reply:
yes because of the linters, and they are our dev dependencies, so in my opinion it is correct